xla/pjrt/gpu/BUILD

```diff
@@ -153,7 +153,7 @@ xla_cc_test(
     srcs = if_gpu_is_configured(["se_gpu_pjrt_client_test.cc"]),
     tags = [
         "gpu",
-        "no_oss",
+        # "no_oss",
```

Comment:
Is it accidental?

Reply:
Sorry. Accident. I need to test so I comment it out. 
Unfortunately now I am falling into this category...